package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mosip-resident-app",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Please update your node/npm, remove node_modules and package-lock.json, then `npm install`